website/public/js/controllers/notificationCtrl.js

```diff
@@ -53,6 +60,23 @@ habitrpg.controller('NotificationCtrl',
        Notification.mp(mana);
     });
 
+    $rootScope.$watch('user.stats.lvl', function(after, before) {
+      if (after <= before) return; 
+      Notification.lvl();
+      $rootScope.playSound('Level_Up');
+      if (User.user._tmp && User.user._tmp.drop && (User.user._tmp.drop.type === 'Quest')) return;
+      if (after === 3) return; // Drop system unlock. FIXME can we do this without hardcoding?
```

Comment:
OK. Right in the same controller, then? I guess that would be easier to update, though it'd still be nice to have the milestone list in a more central place.